TF_Application/Assets/TouchFree_Application/Scripts/TransparentWindow.cs

```diff
@@ -62,7 +62,7 @@ void Start()
         {
 #if !UNITY_EDITOR // You really don't want to enable this in the editor..
             hwnd = FindWindow(null, "TouchFree");
-            SetLayeredWindowAttributes(hwnd, 0, 255, LWA_ALPHA);// Transparency=51=20%
+            //SetLayeredWindowAttributes(hwnd, 0, 255, LWA_ALPHA);// Transparency=51=20%
```

Comment:
👀 we should remove this line instead of commenting it out (it's in git history if we do end up needing it)